src/components/Header.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div class="container">
-    <h1 class="title">To do list</h1>
-    <button @click="addTodo" class="add"><img :src="Addplus" /></button>
+    <div class="header">
+      <h1 class="title">To do list</h1>
+      <button @click="addTodo" class="add"><img :src="Addplus" /></button>
```

Comment:
This should act like a real button please consider the different states too (active, hover, disable) and for the + sign you can use an icon library like [Heroicons](https://heroicons.com/)